Spigot-Server-Patches/0619-Added-SoundBuilder.patch

```diff
@@ -0,0 +1,438 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fruxz <cedricspitzer@outlook.de>
+Date: Wed, 30 Dec 2020 02:23:06 +0100
+Subject: [PATCH] Added SoundBuilder
+
+
+diff --git a/src/main/java/io/papermc/paper/sound/PaperSoundBuilder.java b/src/main/java/io/papermc/paper/sound/PaperSoundBuilder.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..236c6ca30fa8d45926900985627d24f01b808e01
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/sound/PaperSoundBuilder.java
+@@ -0,0 +1,368 @@
++package io.papermc.paper.sound;
++
++import com.google.common.collect.Lists;
++import net.minecraft.server.BaseBlockPosition;
++import net.minecraft.server.MinecraftKey;
++import net.minecraft.server.PacketPlayOutCustomSoundEffect;
++import net.minecraft.server.Vec3D;
++import org.bukkit.Location;
++import org.bukkit.Sound;
++import org.bukkit.SoundCategory;
++import org.bukkit.World;
++import org.bukkit.craftbukkit.entity.CraftPlayer;
++import org.bukkit.entity.Player;
++import org.bukkit.util.NumberConversions;
++
++import java.util.Collection;
++import java.util.List;
++
++/**
++ * Helps prepare a sound to be sent to players.
++ */
```

Comment:
In this implementation class you copy-pasted the interface JavaDocs for some reason. Don't do that. When you override the method you don't have to copy-paste the JavaDocs.